Navigation/index.js

```diff
@@ -0,0 +1,3 @@
+import AppNavigator from './Navigation';
```

Comment:
`export { default as AppNavigator } from "./Navigation";` for consistency with other index.js files.